sklearn/linear_model/logistic.py

```diff
@@ -1568,6 +1577,15 @@ class LogisticRegressionCV(LogisticRegression, BaseEstimator,
         Actual number of iterations for all classes, folds and Cs.
         In the binary or multinomial cases, the first dimension is equal to 1.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import load_iris
+    >>> from sklearn.linear_model import LogisticRegressionCV
+    >>> X, y = load_iris(return_X_y=True)
+    >>> clf = LogisticRegressionCV(cv=3, random_state=0).fit(X, y)
```

Comment:
I'd rather cv=5. We don't recommend cv=3

Reply:
That'll make the `score` of `LogisticRegressionCV` to be lower than `LogisticRegression`. That's why @qinhanmin2014 suggested to lower the `cv` to 3. I can take it back to 5, shall I do that and ignore the lower score?

Reply:
Yes, it should be lower.

Reply:
Done.